packages/openid4vc-client/src/utils/metadata.ts

```diff
@@ -0,0 +1,70 @@
+import type { W3cCredentialRecord } from '@aries-framework/core'
+import type {
+  CredentialIssuerMetadata,
+  CredentialsSupportedDisplay,
+  CredentialSupported,
+  EndpointMetadata,
+  EndpointMetadataResult,
+  IssuerCredentialSubject,
+  IssuerMetadataV1_0_08,
+  MetadataDisplay,
+} from '@sphereon/oid4vci-common'
+
+export interface OpenId4VcCredentialMetadata {
+  credential: {
+    display?: CredentialsSupportedDisplay[]
+    order?: string[]
+    credentialSubject: IssuerCredentialSubject
+  }
+  issuer: {
+    display?: MetadataDisplay[]
+    id: string
+  }
+}
+
+// what does this mean
+const openId4VcCredentialMetadataKey = '_paradym/openId4VcCredentialMetadata'
```

Comment:
This is a storage key that can be used to retrieve metadata about the credential. Likely for ease of access for displaying. @TimoGlastra likely knows whether we have to keep an equivalent or if this is up to the user to define.

Reply:
The PR is not yet finished. Just pushed some progress

Reply:
This is the display / issuer metadata. I chose to not add it as PR in AFJ as the spec is continuously evolving and thus it will contain different structures in the metadata. Not sure if that's a problem. It is very useful to have this though, but it may be better if we can generalize it and create a universal credential display format (not sure what OCA uses for this) with something like https://identity.foundation/wallet-rendering/